examples/neural_sde_gan.ipynb

```diff
@@ -725,7 +728,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.10.4"
```

Comment:
Can the spurious changes to this file be removed? You don't need to actually re-run a notebook when you just make changes to the documentation, as it's just a big JSON file you can edit.